metadata/latest/schema.json

```diff
@@ -61,239 +73,296 @@
-                "title": "Subject"
+                "badge": "Platinum",
+                "title": "Subject",
+                "additionalProperties": false
             },
+            "badge": "Platinum",
             "title": "Subject"
         },
         "language": {
-            "description": "Language used within the described data structures (e.g. titles, descriptions). The language key can be repeated if more languages are used. Standard: IETF (BCP47). Example: [en-GB, de-DE, fr-FR]",
+            "description": "List of languages used within the described data structures (e.g. titles, descriptions). Standard: IETF (BCP47).",
+            "example": "[en-GB, de-DE, fr-FR]",
             "type": "array",
             "items": {
+                "description": "Language used within the described data structures (e.g. titles, descriptions). The language key can be repeated if more languages are used. Standard: IETF (BCP47)",
+                "example": "en-GB",
                 "type": [
                     "string",
                     "null"
                 ],
+                "badge": "Gold",
                 "title": "Language"
             },
+            "badge": "Gold",
             "title": "Language"
         },
         "keywords": {
-            "description": "An array of string keywords to assist users searching for the package in catalogs. Example: [example, template, test]",
+            "description": "A list of string keywords to assist users searching for the package in catalogs.",
+            "example": "[example, template, test]",
             "type": "array",
             "items": {
+                "description": "A keyword to assist users searching for the package in catalogs.",
+                "example": "template",
                 "type": [
                     "string",
                     "null"
                 ],
+                "badge": "Silver",
                 "title": "Keyword"
             },
-            "title": "Keyword"
+            "badge": "Silver",
+            "title": "Keywords"
         },
         "publicationDate": {
-            "description": "Date of publishing. Date Format is ISO 8601 (YYYY-MM-DD). Example: 2019-02-06",
+            "description": "Date of publishing. Date Format is ISO 8601 (YYYY-MM-DD)",
+            "example": "2019-02-06",
             "type": [
                 "string",
                 "null"
             ],
+            "badge": "Bronze",
             "title": "Publication date",
             "format": "date"
         },
         "context": {
-            "description": "Object. Contains name-value-pairs that describe the general setting, evironment or project leading to the creation or maintenance of this dataset.",
+            "description": "An Object that describes the general setting, evironment or project leading to the creation or maintenance of this dataset. In science this can be the research projecti.",
```

Comment:
```suggestion
            "description": "An Object that describes the general setting, evironment or project leading to the creation or maintenance of this dataset. In science this can be the research project.",
```